build.sbt

```diff
@@ -39,18 +42,18 @@ def commonProject(id: String): Project = {
 
 def sttpPlayJson(includePlayVersion: String, scalaVersion: String): Project = {
   val (playSuffix, crossScalaVersion) = includePlayVersion match {
-    case Dependencies.play23Version => ("23", CrossScalaVersion)
-    case Dependencies.play25Version => ("25", CrossScalaVersion)
-    case Dependencies.play26Version => ("26", Seq.empty)
+    case Dependencies.play23Version => ("23", Seq.empty)
```

Comment:
Did you mean to add 2.4 too?

Reply:
No I mean you didn't include 2.4 in the list, but you list it in the PR title. is that intentional?

Reply:
I overlooked that :)
I'll try to make it work with your idea.